membership-attribute-service/app/controllers/AccountController.scala

```diff
@@ -321,9 +335,6 @@ class AccountController(
     } yield validAmount
   }
 
-  def cancelSpecificSub(subscriptionName: String): Action[AnyContent] =
```

Comment:
these last few wrapper methods aren't needed any more as the called method now takes a string directly